### ISSUE TITLE
Bug: Getting Error: Bad credentials when running ZAP full scan in github action

### ISSUE THREAD

**Issue by ayushgarg3107** [zaproxy/action-full-scan#39]
Error logs 

```
2022-05-17 06:54:46,035 Starting new HTTP connection (1): localhost:56760
2022-05-17 06:54:46,296 http://localhost:56760 "GET http://zap/OTHER/core/other/mdreport/?apikey= HTTP/1.1" 200 10977
FAIL-NEW: 0	FAIL-INPROG: 0	WARN-NEW: 2	WARN-INPROG: 0	INFO: 0	IGNORE: 0	PASS: 112
2022-05-17 06:54:46,305 Trigger hook: zap_pre_shutdown, args: 1
2022-05-17 06:54:46,307 Starting new HTTP connection (1): localhost:56760
2022-05-17 06:54:46,349 http://localhost:56760 "GET http://zap/JSON/core/action/shutdown/?apikey= HTTP/1.1" 200 15
2022-05-17 06:54:46,351 Trigger hook: pre_exit, args: 3
[@octokit/rest] `const Octokit = require("@octokit/rest")` is deprecated. Use `const { Octokit } = require("@octokit/rest")` instead
Scanning process completed, starting to analyze the results!
Error: Bad credentials
```

Full ZAP action Logs 

[ZAP_github_action_raw_logs.txt](https://github.com/zaproxy/action-full-scan/files/8726139/ZAP_github_action_raw_logs.txt)

Github action flow file.

```yaml
zap_scan:
    runs-on: self-hosted
    name: Scan using ZAP
    steps: 
      - name: 'Git Checkout'
        uses: actions/checkout@v2

      - name: ZAP Scan
        uses: zaproxy/action-full-scan@v0.3.0
        with:
          token: ${{ secrets.GHE_TOKEN }}
          docker_name: 'owasp/zap2docker-stable'
          target: 'https://xxxxxxxxx'
          cmd_options: '-d'
```
          
         
Thanks in advance

**Comment by psiinon** [comment]
https://mmmazurepetstore.azurewebsites.net/ appears to be protected by Google SSO.
Have you configured your action to avoid that in some way?
Its not something the action will be able to do on its own...

**Comment by ayushgarg3107** [comment]
@psiinon Why is the scan working with the docker image in the local machine without any issues?

**Comment by psiinon** [comment]
I dont know, I dont have access to the target app or your local machine :D 
Can you access your target app without SSO from your local machine?

**Comment by thc202** [comment]
Closing, this is now fixed in latest version (v0.8.0). Fixed by zaproxy/actions-common#42.
